lib/customisation.nix

```diff
@@ -142,13 +142,22 @@ rec {
       outputToAttrListElement = outputName:
         { name = outputName;
           value = commonAttrs // {
-            inherit (drv.${outputName}) outPath drvPath type outputName;
+            inherit (drv.${outputName}) type outputName;
+            drvPath = assert condition; drv.${outputName}.drvPath;
+            outPath = assert condition; drv.${outputName}.outPath;
           };
         };
 
       outputsList = map outputToAttrListElement outputs;
-  in commonAttrs // { outputUnspecified = true; };
+    in commonAttrs // {
+      outputUnspecified = true;
+      drvPath = assert condition; drv.drvPath;
+      outPath = assert condition; drv.outPath;
```

Comment:
follow-up: I think exposing the `condition` here might be helpful in the future as a way to filter out bad packages from the input lists, such as preventing X libraries to be used.

Note, In the case of `check-meta.nix`, this `assert condition;` statement is only used to enforce the strictness of the condition, as it will throw before the assertion can catch it. There is no proper `false` value returned, unless the `config.handleEvalIssue` function is set to return it.

Reply:
> `check-meta.nix` will throw before the assertion can catch it

`check-meta.nix` is being changed in this PR to return a boolean.